index.html

```diff
@@ -424,7 +424,67 @@ <h3>Terminal</h3>
                     </ul>
                 </div>
             </section>
+            <section class="contactme">
+                <div class="contactme__content">
+                    <div class="contactme__details">
+                        <p>
+                            I'm always <br> interested in hearing <br>
+                            about new projects, <br>
+                            so if you'd like to <br>
+                            chat please get in touch.
+                        </p>
+                    </div>
+                    
+                </div>
+                <form action="https://formspree.io/f/mzbqzrnz" method="POST" class="contactme__content">
+                    <div class="contactme__formgroup">
+                        <input type="text" placeholder="Full Name" class="contactme__input" required maxlength="30">
+                    </div>
+                    <br>
+                    <div class="contactme__formgroup">
+                        <input type="email" placeholder="Email" class="contactme__input" required maxlength="30">
+                    </div>
+                    <br>
+                    <textarea name="contactme" id="" cols="30" rows="10" placeholder="Enter text here" class="contactme__textarea" required maxlength="500"></textarea>
+                    <br>
+                    <button class="contactme__submit-button" type="submit">Get In Touch</button>
+                </form>
+            </section>
+
         </main>
+
+        <footer>
+            <div class="footer__container">
+                <ul>
+                    <li>
+                        <a href="#">
+                            <img src="./images/footer/github.png" alt="no internet">
+                        </a>
+                    </li>
```

Comment:
- [ ] **[OPTIONAL]:** It would be great if you could add your social media links to each one of these icons. Remember that the purpose of these elements is to make it easier for people (future recruiters) to easily connect with you! 🤓 🔧